src/components/wizard/wizard.component.ts

```diff
@@ -353,15 +353,17 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
      * If a step in the wizard becomes invalid, all steps sequentially after
      * it, should become unvisited and incomplete
      */
-    setNextStepsUnvisited(currentStep: WizardStepComponent, cb?: (step: WizardStepComponent | MarqueeWizardStepComponent) => void): void {
+    protected _setNextStepsUnvisited(currentStep: WizardStepComponent, cb?: (step: WizardStepComponent | MarqueeWizardStepComponent) => void): void {
```

Comment:
We can remove the underscore prefix here. There are only two times we prefix things with an underscore:

1) Private class variables. Often class variables will have public setters/getters to to retain the same name the setter would be called `valid` and the private variable would be `_valid` keeping the association very obvious. To be consistent and support any future setter/getter we prefix all private variables with an underscore.

2) Methods that should be private or protected but can't be because they are used within a template.

In this case the function is marked as protected so typescript will prevent anyone from calling it, so we can omit the underscore prefix.